sepal_ui/bin/module_venv

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python3
+
+"""
+Script to add a testing kernel in the local environment
+
+This script should be run in a module folder. From there, it will create a  new venv in the "module_venv" folder.
+This venv will be named after the module repository name. All librairies specified in the requirements.txt file will be installed.
+Then the script will read the "module.yaml" file (if existing) and exctract, the name of the module and the entry_point file (ususally "ui.ipynb").
+The venv will be added to the list of usable Jupyter kernel. It will be named: "test-<name_of_the_repo>" and displayed as "(test) <name of the module>".
+The kernel should automatically be added to the entry_point file.
+"""
+
+from pathlib import Path
+from shutil import rmtree
+import subprocess
+import yaml
+import json
+import argparse
+
+from colorama import init, Fore
+
+# init colors for all plateforms
+init()
+
+if __name__ == "__main__":
+
+    # read arguments (there should be none)
+    p = argparse.ArgumentParser(usage=__doc__)
+    args = p.parse_args()
+
+    # welcome the user
+    print(f"{Fore.YELLOW}venv creation interface{Fore.RESET}")
+
+    # create a venv folder
+    print('create the venv directory: "module-venv"')
+    venv_dir = Path.home() / "module-venv"
+    venv_dir.mkdir(exist_ok=True)
+
+    # create a venv folder associated with the current repository
+    print(f'create a venv directory for the current app: "{Path.cwd().stem}"')
+    current_dir_venv = venv_dir / Path.cwd().stem
+
+    # empty the folder from anything already in there
+    # equivalement to flushing the existing venv (it's just faster than pip)
+    if current_dir_venv.is_dir():
+        rmtree(current_dir_venv)
+
+    current_dir_venv.mkdir(exist_ok=True)
+
+    # init the venv
+    print("init the venv...")
+
+    cmd = ["python3", "-m", "venv", str(current_dir_venv)]
+    res = subprocess.run(cmd, cwd=Path.cwd())
+
+    pip = str(current_dir_venv / "bin" / "pip")
+
+    cmd = [pip, "install", "wheel"]
+    res = subprocess.run(cmd, cwd=Path.cwd())
+
+    cmd = [pip, "install", "Cython"]
+    res = subprocess.run(cmd, cwd=Path.cwd())
+
+    cmd = [pip, "install", "ipykernel"]
+    res = subprocess.run(cmd, cwd=Path.cwd())
+
+    # install all the requirements
+    cmd = [pip, "install", "-r", str(Path.cwd() / "requirements.txt")]
```

Comment:
Can we merge the cmd's and loop the `run` method? perhaps we can save some lines :)